packages/mobile/src/account/selectors.ts

```diff
@@ -12,3 +12,7 @@ export const getOutgoingPaymentRequests = (state: RootState) => {
     (p) => p.status === PaymentRequestStatus.REQUESTED
   )
 }
+
+export const getE164PhoneNumber = (state: RootState) => {
```

Comment:
Dupe of https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/account/reducer.ts#L195

Reply:
Maybe we should move selectors out of the reducer files to avoid this confusion

Reply:
I think it's a good idea, will move them there 👍